test/serverTest.js

```diff
@@ -83,7 +84,10 @@ for (const supportedVersion of mc.supportedVersions) {
         senderTeam: undefined,
         timestamp: Date.now(),
         salt: 0n,
-        signature: Buffer.alloc(0)
+        signature: mcData.supportFeature('useChatSessions') ? undefined : Buffer.alloc(0),
```

Comment:
useChatSessions does not exist

Reply:
nevermind

Reply:
https://github.com/PrismarineJS/minecraft-data/blob/1f1eec2ba642915aca4fec76e8bbe086bd6d73b6/data/pc/common/features.json#L691

Reply:
why is signature undefined if it exists ?

Reply:
If we are in 1.19.3 we don't want to send the signature field